OsmAnd/res/values/strings.xml

```diff
@@ -30,19 +30,19 @@
     <string name="subscription_on_hold_title">OsmAnd Live subscription is on hold</string>
     <string name="subscription_paused_title">OsmAnd Live subscription has been paused</string>
     <string name="subscription_expired_title">OsmAnd Live subscription has been expired</string>
-    <string name="subscription_payment_issue_title">There is a problem with your subscription. Click the button to go to the Google Play subscription settings to fix your payment method.</string>
+    <string name="subscription_payment_issue_title">Click this button to set up a payment method on Google Play to fix your subscription.</string>
     <string name="manage_subscription">Manage subscription</string>
     <string name="user_login">Login</string>
     <string name="user_password">Password</string>
     <string name="login_account">Account</string>
     <string name="use_login_password">Use login and password</string>
     <string name="open_street_map_login_mode">You need to login to upload new or modified changes. \n\nYou can log in using the safe OAuth method or use your login and password.</string>
-    <string name="osm_edits_view_descr">You can view all your not yet uploaded edits or OSM bugs in %1$s. Uploaded points don’t show in OsmAnd.</string>
+    <string name="osm_edits_view_descr">All your not yet uploaded edits or OSM bugs can be viewed in %1$s. Uploaded points don’t show up in OsmAnd.</string>
     <string name="sign_in_with_open_street_map">Sign in with OpenStreetMap</string>
     <string name="login_open_street_map_org">Login to OpenStreetMap.org</string>
     <string name="login_open_street_map">Login to OpenStreetMap</string>
     <string name="plugin_global_prefs_info">These plugin setting are global, and apply to all profiles</string>
-    <string name="message_you_need_add_two_points_to_show_graphs">You need to add at least two points</string>
+    <string name="message_you_need_add_two_points_to_show_graphs">Add at least two waypoints</string>
```

Comment:
?

Reply:
We use point and waypoint in differing meanings, I think.

Reply:
If so, what kind of points are "points"?

Reply:
`<trkpt>`, I believe.